docs/src/General/noteworthy-differences.md

```diff
@@ -0,0 +1,30 @@
+# Noteworthy Differences from other computer algebra systems
```

Comment:
This could also be
```suggestion
# Notes for users from other computer algebra systems
```


We could then also tell e.g. a Singular/GAP/Polymake user's how to access features from "their" systems from Oscar, even if Oscar does not yet have them.